packages/expo/scripts/aggregate_xc_privacy.rb

```diff
@@ -0,0 +1,48 @@
+require 'plist'
+
+def aggregate_xc_privacy!(installer)
+  mapping = {}
+  project = installer.aggregate_targets.first.user_project
+  installer.pod_targets.each do |pod_target|
+    pod_target.file_accessors.each do |file_accessor|
+      file_accessor.resource_bundles.each do |bundle_name, bundle_files|
+        bundle_files.each do |file_path|
+          if File.basename(file_path) == 'PrivacyInfo.xcprivacy'
+            content = Plist.parse_xml(file_path)
+            accessed_api_types = content["NSPrivacyAccessedAPITypes"]
+            accessed_api_types.each do |accessed_api|
+              api_type = accessed_api["NSPrivacyAccessedAPIType"]
+              reasons = accessed_api["NSPrivacyAccessedAPITypeReasons"]
+              
+              mapping[api_type] ||= []
+              mapping[api_type] += reasons
+            end
+          end
+      
+        end
+      end
+    end
+  end
+
+  # Get a reference to the Xcode project
+  project = installer.aggregate_targets.first.user_project
```

Comment:
Not sure if using `first` here is a good idea. We should at least find the target that is an app target (not an app or watchOS extension).

Reply:
Will add a comment, but from my understanding this structure is due to the fact that cocoapods can install deps for multiple projects at once, but in our case `user_project` should be the same for all `aggregate_targets`